tests/kerastuner/mock_distribute.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 The Keras Tuner Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Mock running KerasTuner in a distributed tuning setting."""
+
+import multiprocessing
```

Comment:
Multiprocessing won't work on Borg (cf long string of issues we've had with Keras multiprocessing utilities) and causes frequent hanging on Travis as well (which are by nature hard to debug). Could we use threading instead?

Reply:
Done